### ISSUE TITLE
Hi Adrian, I have a question on importing styles in Contact.jsx.

### ISSUE THREAD

**Issue by kohta9521** [adrianhajdin/project_3D_developer_portfolio#20]
Thanks Adrian for all your great videos.

I have one ERROR happening in the part of App.jsx that imports each component, specifically this code.
The imoprt statement in the Contact.jsx file says it can't find the styles file, and pasting the Github code doesn't work, so I need your help!

```
[plugin:vite:import-analysis] Failed to resolve import "../styles" from "src/components/Works.jsx". Does the file exist?
/Users/?????/Desktop/Portfolio/src/components/Works.jsx:5:23
17 |  import Tilt from "react-tilt";
18 |  import { motion } from "framer-motion";
19 |  import { styles } from "../styles";
   |                          ^
20 |  import { github } from "../assets";
21 |  import { SectionWrapper } from "../hoc";
```

I'm not sure if the file itself exists, because I've had no errors with the styles import on other components.

I live in Japan and my English is not very good, so there might be something wrong with my writing.
I'm sorry if I'm wrong, but please help me.

**Comment by Alecfut07** [comment]
I have the same error as you, but in my case, I had a file called "style.js" inside the src folder. So I just need to rename the file to "styles.js".

**Comment by kohta9521** [comment]
Thanks for the answer!
I changed style.js to styles.js as you taught me and the error disappeared the first time. However, the next time I imported the SectionWrappern section, the error appeared.

```
20 |  import { styles } from "../styles";
21 |  import { experiences } from "../constants";
22 |  import { SectionWrapper } from "../hoc";
   |                                  ^
23 |  import { textVariant } from "../utils/motion";
24 |  const ExperienceCard = ({
```


Did you get the same error?

**Comment by Alecfut07** [comment]
Yeah, I also got the same error, but I already fixed it. You need to do this in index.js and SectionWrapper.jsx from hoc folder:
![index_js](https://user-images.githubusercontent.com/42727555/224597907-adea98a7-280d-4fd7-8e19-bd687ab34936.png)
![SectionWrapper_jsx](https://user-images.githubusercontent.com/42727555/224597909-cb854d8f-6242-48f9-9afb-b4c1ce5dc5ea.png)
